src/main/java/net/minecraftforge/registries/ForgeRegistry.java

```diff
@@ -871,6 +883,45 @@ public Snapshot makeSnapshot()
         return ret;
     }
 
+    private class RegistryCodec implements Codec<V>
+    {
+        @Override
+        public <T> DataResult<Pair<V, T>> decode(DynamicOps<T> ops, T input)
+        {
+            if(ops.compressMaps())
```

Comment:
Formatting: space before opening parenthesis.
```suggestion
            if (ops.compressMaps())
```